index.html

```diff
@@ -4853,6 +4853,74 @@ <h3>Issuer</h3>
+Often relevant metadata about the <a>holder</a>, as identified by the value of the
+<code>holder</code> <a>property</a>, is available to, or
+retrievable by, the <a>verifier</a>. For example, a <a>holder</a> can
+publish information containing the verification material used to secure
+<a>verifiable presentations</a>. This metadata is expected to be used when
+checking proofs on <a>verifiable presentations</a>. Some cryptographic
+identifiers contain all necessary metadata in the identifier itself. In those 
+cases, no additional metadata is required. Other identifiers use verifiable
+data registries where such metadata is automatically published for use
+by <a>verifiers</a>, without any additional action by the <a>holder</a>.
+        </p>
+        <p>
+See the <a data-cite="VC-IMP-GUIDE/#subject-holder-relationships"></a> and
+<a data-cite="VC-USE-CASES#user-tasks"></a> for additional examples related to <a>subject</a> and <a>holder</a>.
+        </p>
+
+        <p class="note">
+`Issuer`, `subject`, and `holder` are graph nodes which support multiple representations, 
+potentially making it complex to evaluate whether these roles are being filled
+by appropriate entities.  Validation is the process by which verifiers apply business rules to 
+evaluate the propriety of a particular use of a <a>verifiable credential</a>.
+        </p>
+        <ul>
+          <li>
+<a href="#issuer">Issuers</a> define expressions of an `issuer` in <a>credentials</a>
+          </li>
+          <li>
+<a href="#presentations-0">Presentations</a> define expressions of a `holder` in <a>presentations</a>
+          </li>
+          <li>
+<a href="#credential-subject">Credential subjects</a> define expressions of a `credentialSubject` in <a>credentials</a>
+          </li>
+        </ul>
+        <p>
+A <a>verifier</a> might need to validate a given <a>verifiable presentation</a>
+against complex business rules; for example, the verifier might need confidence
+that the <a>holder</a> is the same entity as a <a>subject</a> of a <a>verifiable
+credential</a>. In that situation, the following factors can provide
+reasonable confidence that the holder presenting a given 
+<a>verifiable credential</a> is, in fact, a subject of that 
+<a>verifiable credential</a>:
+        </p>
+        <ul>
+          <li>
+The <a>verifiable presentation</a> is secured, 
+using a mechanism the <a>verifier</a> trusts to protect the integrity of the content.
+          </li>
+          <li>
+The <a>verifiable presentation</a> includes one or more <a>verifiable credentials</a> that are secured, 
+using a mechanism the <a>verifier</a> trusts to protect the integrity of the content.
+          </li>
+          <li>
+When the identifiers for `holder` and `subject` are the same.
+          </li>
+          <li>
+When the verification material used to secure a <a>verifiable presentation</a>
+is also present in the claims about the credential <a>subject</a>, either by value or by reference.
```

Comment:
This seems incorrect.

As long as the verifier can retrieve the appropriate verification material, it need not be present in the claims. Although you can infer that "by reference" with the right logical gymnastics, it's more clear to say that the claims in a credentialSubject value are about the subject referred to by the identifier of that entry in the credentialSubject array. The "id" is not a claim, nor is it "present in the claims".

```suggestion
When that common identifier can be used to discover or derive the verification material used to verify the integrity of that<a>verifiable presentation</a>.

When these are true, then <a>verifiers</a> can reasonably infer that the claims expressed regarding that identifier, in included VCs, are about the current presenter.
```

It's important that we respect the meaning of this in the case of multiple subjects. In a marriage license, the officiant would be identified and they are as legitimate a holder/presenter as the witness or the spouses.

Reply:
Tweaks to @jandrieu's suggestion. Note that @jandrieu's second paragraph here is redundant against lines 4900-4903 above, which introduced this `<ul>`; 
```suggestion
When that common identifier can be used to discover or derive the verification
material used to verify the integrity of that <a>verifiable presentation</a>.
```

Reply:
@jandrieu I took @TallTed 's suggestion feel free to suggest further corrections now that its been applied.